### DISCUSSION TITLE
账户余额2.8美元但是报错"Invalid URL (GET /v1/chat/completions)"如何解决？

### DISCUSSION THREAD

**Discussion by Huaweidev** [BurnHair/docs#122] [技术支持]
账户余额2.8美元但是报错"Invalid URL (GET /v1/chat/completions)"如何解决？
试用任何模型都是这样：
```
{
  "error": {
    "message": "Invalid URL (GET /v1/chat/completions)",
    "type": "invalid_request_error",
    "param": "",
    "code": ""
  }
}

```

**Comment by BennyThink** [comment]
https://github.com/BurnHair/docs/blob/master/lobe-chat%20%E5%92%8C%20ChatGPT-Next-Web%20%E4%BD%BF%E7%94%A8.md#%E9%85%8D%E7%BD%AE-1

已经email回复你了。
